resources/lang/front/nl/form.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+return [
+
+    'validation_error.title' => 'Je invoer is invalide.',
+    'validation_error.description' => 'Controlleer uw invoer, fouten worden weergegeven onder ieder veld.',
+
+    'list.remove_button' => 'Verwijderen',
+    'list.insert_button' => 'Invoegen',
+    'list.sort_button.inactive' => 'Sorteren',
+    'list.sort_button.active' => 'Voltooi',
+    'list.empty' => 'Lege lijst',
+
+    'geolocation.loading' => 'Google Maps is aan het laden...',
+    'geolocation.browse_button' => 'Kies een locatie...',
+    'geolocation.remove_button' => 'Verwijderen',
+    'geolocation.edit_button' => 'Bewerken',
+    'geolocation.modal.title' => 'Typ een adres or klik op de kaart',
+    'geolocation.modal.title-no-geocoding' => 'Klik op de kaart',
+    'geolocation.modal.geocode_input.placeholder' => 'Volledig adres',
+    'geolocation.modal.geocode_input.message.error' => "Foutmelding: Kan geen verbindingen maken met de Geocoding server",
+    'geolocation.modal.geocode_input.message.no_results' => 'Geen resultaten voor ":query"',
+    'geolocation.modal.search_button' => 'Zoeken',
+
+    'upload.browse_button' => 'Bladeren...',
+    'upload.remove_button' => 'Verwijderen',
+    'upload.edit_button' => 'Bewerken',
+    'upload.download_link' => 'Download',
+    'upload.message.file_too_big' => 'Het geselecteerde bestand is te groot',
+    'upload.message.bad_extension' => 'De bestands extensie is niet valide',
```

Comment:
Can also change 'niet valide' to 'onjuist' everywhere in this fine. The former sounds artificial and non-natural imho